### ISSUE TITLE
Unable to install both elm-ui and elm-css

### ISSUE THREAD

**Issue by kwrooijen** [gdotdesign/elm-ui#61]
When adding both elm-ui and elm-css to my dependencies I get the following error:

````
The elm-package.json constraints of 'gdotdesign/elm-ui' are probably
letting too much stuff through. Definitely open an issue on the relevant github
repo to get this fixed and save other people from this pain.

In the meantime, take a look through the direct dependencies of the broken
package and see if any of them have had releases recently. If you find the new
thing that is causing problems, you can artificially constrain things by adding
some extra constraints to your elm-package.json as a stopgap measure.
````

However I think the issue is that both `elm-ui` and `elm-css` have a module named `Css`.
I see that this project has its own Css solution, but my project is already quite invested in `elm-css`, and I just wanted to use `elm-ui` for the HTML components.

Is there any way I can use both `elm-ui` and `elm-css` ?

**Comment by gdotdesign** [comment]
I've run into this problem myself recently, and was surprised that they can clash.

The easiest solution here is to move the `Css` and `Css.Properties` modules under the `Ui` namespace (as `Ui.Css` and the `Ui.Css.Properties`), since they are not exposed. I don't know when will I have the time to do it though.

**Comment by kwrooijen** [comment]
So if someone took the time to move / rename these modules under `Ui` you'd be willing to merge a PR if approved?

**Comment by gdotdesign** [comment]
Yes, absolutely :+1:

**Comment by gdotdesign** [comment]
Released as 1.0.1 https://github.com/gdotdesign/elm-ui/releases/tag/1.0.1 thanks for your contribution :+1:
